public/css/vpn-promo-banner.css

```diff
@@ -150,4 +160,7 @@ body {
     top: 0.25rem;
     right: 0.25rem;
   }
+  .vpn-promo-banner a {
+    flex-shrink: 0;
+  }
```

Comment:
Added to adjust the design better for non-English languages. It allows the button copy to stay on one-line when on larger screens.